### PR TITLE
More logging

### PR DESCRIPTION
**Pull request by slohse**
**What problems does this PR solve?**
Makes the worker part of the service a lot more verbose.
None of the log messages of the worker were written anywhere, because the worker is invoked by redis queue and doesn't get the log configuration of the webservice. I added a somewhat hardcoded configuration.
Additionally the dsmc-logfiles were also missing, because the environment variable configuring them had a typo.

**How has the changes been tested?**
Tested in production on bioscreen.

**Reasons for careful code review**
If any of the boxes below are checked, extra careful code review should be inititated.

  - [ ] This PR contains code that could remove data

### CODE REVIEW

File: archive_verify/workers.py [hunk 1]
```diff
@@ -39,13 +40,13 @@ def _parse_dsmc_return_code(exit_code, output, whitelist):
 
         for warning in warnings:
             if warning not in whitelist:
-                log.debug("A non-whitelisted DSMC warning was encountered. Reporting it as an error!")
+                log.debug("A non-whitelisted DSMC warning was encountered. Reporting it as an error! ('{}')".format(warning))
```
Comment (b97pla) [inline archive_verify/workers.py]: perhaps "error" is a better log level here?

File: archive_verify/workers.py [hunk 2]
```diff
@@ -39,13 +40,13 @@ def _parse_dsmc_return_code(exit_code, output, whitelist):
...
                 return False
 
         log.debug("Only whitelisted DSMC warnings were encountered. Everything is OK.")
         return True
     else:
-        log.info("An uncatched DSMC error code was encountered!")
+        log.info("An uncaught DSMC error code was encountered!")
```
Comment (b97pla) [inline archive_verify/workers.py]: `log.error(...)` here as well?

File: archive_verify/workers.py [hunk 3]
```diff
@@ -104,19 +107,34 @@ def verify_archive(archive, archive_path, description, config):
     dest_root = config["verify_root_dir"]
     dsmc_log_dir = config["dsmc_log_dir"]
     whitelist = config["whitelisted_warnings"]
+
+    now_str = datetime.datetime.now().strftime('%Y-%m-%d_%H-%M')
+    log.setLevel(logging.DEBUG)
+    fh = logging.FileHandler("{}/{}-{}.log".format(dsmc_log_dir, description, now_str))
```
Comment (b97pla) [inline archive_verify/workers.py]: nitpicking of the day: `os.path.join(dsmc_log_dir, "{}-{}.log".format(description, now_str))`

**Review by b97pla — APPROVED** [review]
Nice work and a great improvement! If you'd like to address my comments, feel free, otherwise I'm ok with merging as is.

File: archive_verify/workers.py (hunk 3 above)
Comment (slohse) [reply archive_verify/workers.py]: I don't feel that it is a nitpick, you are entirely correct in pointing that out.
Well this is embarrassing :sweat:

**Pull request merged**